time_execution/backends/elasticsearch.py

```diff
@@ -89,9 +99,12 @@ def write(self, name, **data):
         data["name"] = name
         data["timestamp"] = datetime.utcnow()
 
-        self.client.index(
-            index=self.get_index(),
-            doc_type=self.doc_type,
-            id=None,
-            body=data
-        )
+        try:
```

Comment:
@igbun @jeduden as we discussed it is worth measuring and comparing performance of ElasticSearch backend to Influx in different scenarios. I'm going to tackle that in a separate issue and PR.